xml/article_sap_trento.xml

```diff
@@ -249,6 +249,11 @@ As agreed on https://confluence.suse.com/x/DAEcN on our Trento doc kick off
           <para>
             &t.server; must be able to reach the &t.agent;s hosts via SSH (port TCP/22).
           </para>
+        </listitem>
+         <listitem>
+          <para>
+            &t.server; must be able to reach the Node Exporter in the &t.agent;s hosts (port TCP/9100).
```

Comment:
I'm unsure of the spelling of "Node Exporter". Is this a name or a general thing?

Reply:
It is the technical name of the component that collects data from the host where the agent is running. Check here: https://github.com/prometheus/node_exporter.

Reply:
Ahh, I see. Thanks for the info. I'll merge it.